Clinical-Genomics/scout
# Bugfix #2058 loqusdb feature 200901

This PR fixes a bug. fix #2058

**How to test**:
1. Install branch.
2. In Web-GUI go to Institute settings `http://localhost:5000/overview/cust000/settings?`
3. Delete "LoqusDB id" and save
4. Go to a Variant. This should not crash.


**Expected outcome**:
The functionality should be working. No crash.


**Review:**
